### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by miketaochina** [coder/code-server#2418] [Q&A] [answered]
Hi, I use a Mac, `OSX 10.14.6 (18G87)`
when I installed code-server, after I got 2 'info', I got this error message: '`error listen EADDRINUSE: address already in use 127.0.0.1:8080`' message...what I can do?
and when I go to `http://127.0.0.1:8080/login`, it says '`Check the config file at ~/.config/code-server/config.yaml for the password.`', but I do not find it, or I do not know how to find it, could you please help?

**Comment by oxy** [comment, marked as answer]
The error means that you have something else already running that is using port 8080, and since you're able to find the code server `/login` page, I'm assuming that's `code-server`.

To find the file, try opening Terminal and running `cat ~/.config/code-server/config.yaml` - that should work, though I'm not _too_ familiar with macOS.

**Reply by miketaochina** [reply]
Hi,

I tried `cat ~/.config/code-server/config.yaml` and it works! Thanks a lot!
